src/test/test-creds.c

```diff
@@ -246,7 +246,7 @@ TEST(mime_type_matches) {
         FOREACH_ELEMENT(t, tags) {
                 _cleanup_free_ char *encoded = NULL;
 
-                ASSERT_OK(base64mem(t, sizeof(sd_id128_t), &encoded));
+                ASSERT_OK((int) base64mem(t, sizeof(sd_id128_t), &encoded));
```

Comment:
Ah uh, yeah, please use the correct format string in the macro, rather than casting here.